code/modules/gear_presets/survivors/lv_624/clfship_insert_lv624.dm

```diff
@@ -230,4 +231,4 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat, WEAR_FEET)
 	//pockets
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/tools/synth, WEAR_L_STORE)
-	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/construction/full_barbed_wire, WEAR_R_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/survival/synth/full(new_human), WEAR_R_STORE)
```

Comment:
This has a 50 stack metal in it, not only is this not given anywhere else, you are also giving them a large stack of plasteel, not only is this not documented, this is straight up a buff, also not documented at all